static/js/iframe-common.js

```diff
@@ -82,6 +82,11 @@ export function generateIFrame(domain, queryParam, urlParam) {
   // For dynamic iFrame resizing
   iFrameResize({
     checkOrigin: false,
+    onInit: function(iframe) {
+      iframe.iFrameResizer.sendMessage({
```

Comment:
Should we guard this call by the presence of a `token`? In the case of a normal, non-JWT iFrame integration, would this cause us to be sending empty messages around?

Reply:
yep, good catch